Formula/m/mummer.rb

```diff
@@ -0,0 +1,50 @@
+# Formula for installing the mummer aligner. It must be compiled with GCC from
```

Comment:
This comment can go now that the `fails_with` was added, but it should still declare a CLang dependency.

Reply:
I think you mean a dependency on GCC. That was added.